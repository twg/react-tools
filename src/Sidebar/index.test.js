/* eslint-env jest */
import React from 'react'
import { Sidebar } from './'
import renderer from 'react-test-renderer'

describe('Sidebar', () => {
  it('matches snapshot', () => {
    let component = renderer.create(
      <div
        style={{ height: '300px', width: '600px', border: '1px solid grey' }}
      >
        <Sidebar>
          <div>1</div>
          <div>2</div>
          <div>3</div>
          <div>4</div>
          <div>5</div>
          <div>1</div>
          <div>2</div>
          <div>3</div>
          <div>4</div>
          <div>5</div>
          <div>1</div>
          <div>2</div>
          <div>3</div>
          <div>3</div>
          <div>3</div>
          <div>3</div>
          <div>3</div>
          <div>3</div>
          <div>3</div>
          <div>3</div>
          <div>3</div>
          <div>3</div>
          <div>3</div>
          <div>3</div>
        </Sidebar>
      </div>
    )

    expect(component.toJSON()).toMatchSnapshot()
  })
})
