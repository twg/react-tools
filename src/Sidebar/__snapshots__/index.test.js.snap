// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar matches snapshot 1`] = `
<div
  style={
    Object {
      "border": "1px solid grey",
      "height": "300px",
      "width": "600px",
    }
  }
>
  <div
    className="sidebar"
  >
    <div>
      1
    </div>
    <div>
      2
    </div>
    <div>
      3
    </div>
    <div>
      4
    </div>
    <div>
      5
    </div>
    <div>
      1
    </div>
    <div>
      2
    </div>
    <div>
      3
    </div>
    <div>
      4
    </div>
    <div>
      5
    </div>
    <div>
      1
    </div>
    <div>
      2
    </div>
    <div>
      3
    </div>
    <div>
      3
    </div>
    <div>
      3
    </div>
    <div>
      3
    </div>
    <div>
      3
    </div>
    <div>
      3
    </div>
    <div>
      3
    </div>
    <div>
      3
    </div>
    <div>
      3
    </div>
    <div>
      3
    </div>
    <div>
      3
    </div>
    <div>
      3
    </div>
  </div>
</div>
`;
